src/components/posts/postForm.js

```diff
@@ -34,10 +36,28 @@ export const PostForm = (props) => {
       imageRef.current.value = post.image_url
       contentRef.current.value = post.content
 
-      
```

Comment:
@mgjeffries HEY this is already approved BUT - if you fix this ONE line in the edit mode field assignments it works PERFECT with the new server update branch im going to PR
postForm.js @ line 35 change to 
` categoryRef.current.value = post.category.id` (this is how the category id is accessible in the current representation of a post)

Reply:
Will do!